### PR TITLE
Use snake_case uniformly for all routes.

### PR DESCRIPTION
**Pull request by bartkamphorst**

### CODE REVIEW

File: lib/gollum/public/gollum/javascript/gollum.dialog.js [hunk 1]
```diff
@@ -101,7 +101,7 @@
 
       var id = fieldAttributes.id || 'upload';
       var name = fieldAttributes.name || 'file';
-      var action = fieldAttributes.action || '/uploadFile';
+      var action = fieldAttributes.action || '/upload_file';
```
Comment (dometto) [inline lib/gollum/public/gollum/javascript/gollum.dialog.js]: You could consider moving this file to `gollum.dialog.js` and using the path helper here, too

Comment (dometto) [reply lib/gollum/public/gollum/javascript/gollum.dialog.js]: Correction: move it to `gollum.dialog.js.erb`

**Comment by dometto** [comment]
Code zen! I love how little needs to be changed now that we have the path helpers :)

**Comment by bartkamphorst** [comment]
Absolutely, I was pleasantly surprised by that! Kudos!

File: lib/gollum/public/gollum/javascript/gollum.dialog.js (hunk 1 above)
Comment (bartkamphorst) [reply lib/gollum/public/gollum/javascript/gollum.dialog.js]: Agreed. I'm planning to restructure the uploading code base, though, so I'll do that in another PR if you don't mind.

**Pull request merged**